test.js

```diff
@@ -22,15 +22,15 @@ test('number', macro, {args: ['red', 'bold', '123', '--no-stdin']},
 	chalk.red.bold('123'));
 
 test('template', templateMacro, '{red.bold unicorn}',
-	chalk.reset.red.bold('unicorn'));
+	chalk.red.bold('unicorn'));
 test('template strikethrough', templateMacro, '{red.bold unicorns} are {blue.strikethrough FUN!!!}',
-	chalk.reset.red.bold('unicorns') + chalk.reset(' are ') + chalk.reset.blue.strikethrough('FUN!!!'));
+	chalk.red.bold('unicorns') + chalk(' are ') + chalk.blue.strikethrough('FUN!!!'));
 test('template underline', templateMacro, '{red.bold unicorn {blue.underline dancing}}',
-	chalk.reset.red.bold('unicorn ') + chalk.reset.red.bold.blue.underline('dancing'));
+	chalk.red.bold('unicorn ') + chalk.red.bold.blue.underline('dancing'));
 test('template negation', templateMacro, '{red red {~red normal}}',
-	chalk.reset.red('red ') + chalk.reset('normal'));
+	chalk.red('red ') + chalk('normal'));
 
 test('template escaping #1', templateMacro, '{red hey\\} still red} not red',
-	chalk.reset.red('hey} still red') + chalk.reset(' not red'));
+	chalk.red('hey} still red') + chalk(' not red'));
 test('template escaping #2', templateMacro, '{red hey\\\\} not red',
-	chalk.reset.red('hey\\') + chalk.reset(' not red'));
+	chalk.red('hey\\') + chalk(' not red'));
```

Comment:
The places where you removed `.reset()` and left just `chalk` just need to be bare strings. Calling `chalk(someStr)` returns `someStr` unmodified.